### PR TITLE
Allow asciidoctor to read from uri, update README

### PR DESCRIPTION
**Pull request by jadekler**
- Allow asciidoctor to read from URI (contributing section)
- Add 'building documentation' section
- Add newly-updated README
- Small spelling fixes

### CODE REVIEW

**Comment by jadekler** [comment]
Unfortunately missed these in https://github.com/spring-cloud/spring-cloud-cloudfoundry/pull/9.

File: docs/src/main/asciidoc/README.adoc [hunk 1]
```diff
@@ -6,6 +6,27 @@ include::intro.adoc[]
 
 > NOTE: if you are looking for a way to bind to services then this is the wrong library. Check out the [Spring Cloud Connectors](https://github.com/spring-cloud/spring-cloud-connectors) instead.
 
+= Building documentation
```
Comment (spencergibb) [inline docs/src/main/asciidoc/README.adoc]: There's a building section that we usually include:
```
== Building

include::https://raw.githubusercontent.com/spring-cloud/spring-cloud-build/master/docs/src/main/asciidoc/building.adoc[]
```

That would be where to add the building documentation section.

File: docs/src/main/ruby/generate_readme.sh [hunk 2]
```diff
@@ -18,7 +18,7 @@ file = ARGV[0] if ARGV.length>0
 
 srcDir = File.dirname(file)
 out = "// Do not edit this file (e.g. go instead to docs/src/main/asciidoc)\n\n"
-doc = Asciidoctor.load_file file, safe: :safe, parse: false
+doc = Asciidoctor.load_file file, safe: :safe, parse: false, attributes: 'allow-uri-read'
```
Comment (spencergibb) [inline docs/src/main/ruby/generate_readme.sh]: What does this do?

Comment (jadekler) [inline docs/src/main/ruby/generate_readme.sh]: Allows the include macro to import a file from a URI.

File: docs/src/main/asciidoc/README.adoc (hunk 1 above)
Comment (jadekler) [inline docs/src/main/asciidoc/README.adoc]: Good point. Is this PR under consideration? I'll be happy to add the above if so.

Comment (spencergibb) [inline docs/src/main/asciidoc/README.adoc]: Absolutely!

File: docs/src/main/ruby/generate_readme.sh (hunk 2 above)
Comment (spencergibb) [inline docs/src/main/ruby/generate_readme.sh]: Wonder if this needs to go in our other repos too.

Comment (jadekler) [inline docs/src/main/ruby/generate_readme.sh]: I actually don't recall why I put that in there. I think I got it from spring-boot or sagan. I'll double check to see if it does anything.

File: docs/src/main/asciidoc/README.adoc (hunk 1 above)
Comment (jadekler) [inline docs/src/main/asciidoc/README.adoc]: Replaced with the include tag you specified.

File: docs/src/main/ruby/generate_readme.sh (hunk 2 above)
Comment (jadekler) [inline docs/src/main/ruby/generate_readme.sh]: @spencergibb Just checked, without it `link:https://raw.githubusercontent.com/spring-cloud/spring-cloud-build/master/docs/src/main/asciidoc/contributing.adoc[]` is just text. So it looks like it's required =)

**Comment by jadekler** [comment]
@spencergibb All set I think. Let me know of other changes you'd like.

**Pull request merged**